website/docs/d/iam_reference_action.markdown

```diff
@@ -0,0 +1,27 @@
+---
+subcategory : "Account Management"
+---
+
+# ovh_iam_reference_action (Data Source)
+
+Use this data source to list the IAM action associated with a resource type.
+
+## Example Usage
+
+```hcl
+data "ovh_iam_reference_action" "vps_actions" {
```

Comment:
KO

```
│ The provider terraform.local/local/ovh does not support data source "ovh_iam_reference_action". Did you mean "ovh_iam_reference_actions"?
```

![image](https://github.com/ovh/terraform-provider-ovh/assets/2543381/2b1863fd-4623-4025-aa66-4bbf3e4a0bce)

the correct definition is:
```
data "ovh_iam_reference_actions" "vps_actions" {
    type = "vps"
}
```